.travis.yml

```diff
@@ -68,7 +63,7 @@ script:
   - JACOCO_V=$(mvn help:evaluate -Dexpression=jacoco.version | grep -E '^[0-9.]+')
   - export _JAVA_OPTIONS="$_JAVA_OPTIONS -javaagent:$HOME/.m2/repository/org/jacoco/org.jacoco.agent/$JACOCO_V/org.jacoco.agent-$JACOCO_V-runtime.jar=destfile=$PWD/target/jacoco.exec,output=file,append=true,dumponexit=true"
   - echo $_JAVA_OPTIONS
-  - git clone https://github.com/lhfbc/iri-regression-tests.git
+  - git clone https://github.com/lhfbc/iri-regression-tests.git -b remove_milestone_with_conflux
```

Comment:
why hard code this branch name? If you remove this branch, everyone will get panic.

Reply:
I will pull a new request to just remove it after the modification on regression-tests has been merged.